src/thriveopedia/fossilisation/FossilisedSpecies.cs

```diff
@@ -130,6 +130,25 @@ public static FossilisedSpecies LoadSpeciesFromFile(string fossilName)
         };
     }
 
+    /// <summary>
+    ///   Deletes a fossilised species by its filename.
+    /// </summary>
+    /// <param name="fossilName">The name of the .thrivefossil file (including extension)</param>
+    public static void DeleteFossilFile(string fossilName)
+    {
+        var target = Path.Combine(Constants.FOSSILISED_SPECIES_FOLDER, fossilName);
+
+        using var directory = new Directory();
+
+        if (!directory.FileExists(target))
+            throw new ArgumentException("Fossil with the given name doesn't exist");
+
+        if (directory.Remove(target) != Error.Ok)
+        {
+            throw new IOException("Cannot delete: " + target);
```

Comment:
Try to give a user-friendly error dialog here or a GD.PrintErr() instead of violently throwing an exception here. As a game we want to avoid crashes.